scripts/annotation.js

```diff
@@ -154,9 +139,45 @@ $(document).ready( ()=> {
                 seadragon.hide();
                 image_div.show();
             }
-            image_div.attr('src', '/images?id=' + image);
+            let imgURL = '/images?id=' + image;
+            $.get(imgURL)
+                .done(() => {
+                    // knowing the image exists
+                    image_div.attr('src', imgURL);
```

Comment:
It isn't unnecessary, as it checks to see if it get's a valid response from the server. If it doesn't, it alerts the user. Changing the source without this checking will just result in an improperly loaded image with no information to the user.